ibis_substrait/compiler/translate.py

```diff
@@ -583,6 +583,36 @@ def unbound_table(
     )
 
 
+@translate.register(ops.LocalTable)
+def local_table(
+    op: ops.LocalTable, expr: ir.TableExpr, _: SubstraitCompiler, **kwargs: Any
+) -> stalg.Rel:
+    def path_type_of(name : str | None):
+        if name is None:
+            raise ValueError("local table name is None")
+        if name.startswith("file://"):
+            return "uri_folder" if name.endswith("/") else "uri_file"
+        else:
+            return "uri_path_glob" if name.contains("*") else "uri_path"
+    def format_of(name : str | None):
+        return (stalg.ReadRel.LocalFiles.FileOrFiles.FileFormat.FILE_FORMAT_PARQUET
+                if name.endswith(".parquet")
+                else stalg.ReadRel.LocalFiles.FileOrFiles.FileFormat.FILE_FORMAT_UNSPECIFIED)
+    return stalg.Rel(
+        read=stalg.ReadRel(
+            # TODO: filter,
+            # TODO: projection,
```

Comment:
If you can make an issue for these so we can discuss, that would be great. Ideally we don't have any `TOTO`s without associated issues. Otherwise, if you don't feel like creating tickets please remove the `TODO` and add a small description to the comment.

Reply:
Actually, I [copied](https://github.com/ibis-project/ibis-substrait/blob/94980bcbe5932666ec50698466be369ed5859117/ibis_substrait/compiler/translate.py#L578-L579) these. Presumably there are issues for them that can be referenced?

Reply:
Ah, ok. It's fine to leave them for now